modules/imgproc/src/color.cpp

```diff
@@ -5845,10 +5820,11 @@ static const softdouble D65[] = {softdouble::fromRaw(0x3fee6a22b3892ee8),
                                  softdouble::fromRaw(0x3ff16b8950763a19)};
 
 enum { LAB_CBRT_TAB_SIZE = 1024, GAMMA_TAB_SIZE = 1024 };
-static float LabCbrtTab[LAB_CBRT_TAB_SIZE*4];
+static float *LabCbrtTab;
```

Comment:
1. `const`
2. `= NULL`
```
static const float* LabCbrtTab = NULL;
```
3. alignment requirements?
4. we need to make valgrind happy (suppress these hits). To make it easier, probably we should create new function for memory allocation for such "singleton" objects/buffers.
```
float* LabCbrtTab_ = cv::allocSingleton<float>(LAB_CBRT_TAB_SIZE * 4);
splineBuild(f, LAB_CBRT_TAB_SIZE, LabCbrtTab_); // without static_cast<float*>
LabCbrtTab = LabCbrtTab_;
```